gnocchi/tests/functional/gabbits/resource.yaml

```diff
@@ -312,10 +312,15 @@ tests:
 
     - name: patch resource no data
       desc: providing no data is an error
+      request_headers:
+        accept: application/json
       PATCH: /v1/resource/generic/75C44741-CC60-4033-804E-2D3098C7D2E9
       status: 400
-      response_strings:
-          - "Unable to decode body:"
+      response_json_paths:
+        $.description.cause: "Unable to decode body"
+        $.description.detail:
+          body: []
+          text_type: "Expected object or value"
```

Comment:
$.description.cause: "Unable to decode body"
        $.description.reason: "Expected object or value"